cke-tools/src/Makefile

```diff
@@ -2,7 +2,11 @@ SUDO = sudo
 SUDO_GO = $(SUDO) $(shell which go)
 
 # Run tests
-test:
+check-generate:
+	go mod tidy
+	git diff --exit-code --name-only
+
+test: check-generate
```

Comment:
You should separately run `check-generate` from normal test,
because `check-generate` has side-effects.

@morimoto-cybozu @yokaze

Reply:
@ymmt2005 
I missed that point.
I'll fix it.